src/localizations/es.js

```diff
@@ -0,0 +1,128 @@
+import Link from "../link/component";
+import { ReactComponent as LoveEmoji } from "../empty-workspace/love.svg";
+import React from "react";
+
+const pluralRules = new Intl.PluralRules(`es`);
+
+const localization = {
+  dropHint: () => `Coloque los archivos .attheme o .attheme-editor aquí`,
+
+  error_title: () => `Vaya, parece que ha habido un error.`,
+  error_description: () => <React.Fragment>
+    Envie una captura de pantalla del siguiente error a {}
+    <Link href="//t.me/snejugal" isWhite={true}>
+      el desarrollador en Telegram
+    </Link>
+    {} y describe lo que hiciste para que ocurriera el error. Es posible que también deba enviar el archivo del tema oirginal que utilizó.
+  </React.Fragment>,
+  error_dismiss: () => `Para descartar el error, solo tócalo.`,
+
+  emptyWorkspace_title: () => `Comienza a trabajar en tu tema`,
+  emptyWorkspace_createTheme: () => `Crear nuevo tema`,
+  emptyWorkspace_openTheme: () => `Abrir temas existentes`,
+  emptyWorkspace_credits: () => <React.Fragment>
+    Creador por {}
+    <Link href="//t.me/snejugal">@snejugal</Link>
+    {} y {}
+    <Link href="//t.me/AlexStrNik">@AlexStrNik</Link>
+    {} con {}
+    <LoveEmoji className="emoji" />.
+    {} Verifica {}
+    <Link href="//github.com/snejugal/attheme-editor">
+      el código fuente del editor en Github
+    </Link>
+    {} y suscribete a {}
+    <Link href="//t.me/atthemeeditor">
+      nuestro canal en Telegram
+    </Link>!
+  </React.Fragment>,
+  emptyWorkspace_uploadWaysHint: () => <React.Fragment>
+    También puede arrastrar y soltar {}
+    <Link href="//t.me/atthemeeditorbot">@atthemeeditorbot</Link>
+    {} para abrir los temas a traves de Telegram.
+  </React.Fragment>,
+
+  theme_defaultName: () => `Tema increible`,
+
+  workspace_themeNameLabel: () => `Nombre del tema`,
+  workspace_closeTheme: () => `Cerrar el tema`,
+  workspace_closeThemePrompt: () => `¿Estás seguro de que quieres cerrar el tema?`,
+  workspace_downloadThemeFile: () => `Descargar .attheme directamente`,
+  workspace_createPreview: () => `Crear un vista previa`,
+  workspace_testTheme: () => `Prueba el tema`,
+  workspace_downloadWorkspace: () => `Descargar espacio de trabajo`,
+  workspace_runScript: () => `Ejecuta un script`,
+  workspace_editPalette: () => `Edite la paleta personaliza del tema`,
+  workspace_unaddedVariable: () => `Sin añadir`,
+  workspace_unusedVariable: () => `No utilizado por Telegram`,
+  workspace_obsoleteVariable: () => `Obsoleta`,
+  workspace_nonStandardVariable: () => `No Estándar`,
+  workspace_search: () => `Buscar`,
+  workspace_variablesAmount: ({ total, theme }) => {
+    const forms = {
+      "one one": `${theme} of ${total} variable se agrega al tema`,
+      "one other": `${theme} of ${total} variables se agregan al tema`,
+      "other one": `${theme} of ${total} variable se agrega al tema`,
+      "other other": `${theme} of ${total} las variables se agregan al tema`,
```

Comment:
I'm not a Spanish speaker, but I think these should be reviewed (I guess `se agrega` is lined to `1 of ... variable`. Here are some examples of forms when applied:
`1 of 1 variable is added to the theme`
`1 of 2 variables is added to the theme`
`2 of 1 variables are added to the theme`
`2 of 2 variables are added to the theme`